sos/sosreport.py

```diff
@@ -302,6 +308,12 @@ def __init__(self, args):
                 sys.stderr.write("Unknown preset: '%s'\n" % self.opts.preset)
                 self.preset = self.policy.probe_preset()
                 self.opts.list_presets = True
+
+        # --since depends on --all-logs
+        if self.opts.since is not None and self.opts.all_logs is False:
+            sys.stderr.write("--since can only be used with --all-logs\n")
+            self._exit(1)
```

Comment:
This is a nack from me until `--since` no longer depends on `--all-logs`. Way to common a use case for this restriction.